### PR TITLE
Create 01.en.md

### PR DESCRIPTION
**Pull request by Akashs8383**
hello my name is akash i write tutorial how to install aapnel on hetzner vps my hetzner email is webinhost.in@gmail.com

### CODE REVIEW

File: tutorials/how-to-install-aapanel-on-hetzner-vps/01.en.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/install-apanel-hetzner-vps/en"
```
Comment (ShadowArcanist) [inline tutorials/how-to-install-aapanel-on-hetzner-vps/01.en.md:4]: The actual file path is `tutorials/how-to-install-aapanel-on-hetzner-vps/`

File: tutorials/how-to-install-aapanel-on-hetzner-vps/01.en.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+slug: "install-apanel-hetzner-vps"
+date: "2024-09-22"
+title: "How to Install aPanel on a Hetzner VPS"
```
Comment (ShadowArcanist) [inline tutorials/how-to-install-aapanel-on-hetzner-vps/01.en.md:7]: Small typo - `aaPanel` it's not `aPanel` (the name has 2 "a")

**Comment by ShadowArcanist** [comment]
Hey @Akashs8383 I reviewed your tutorial and I'm unsure if it aligns with the [contributor guidelines](https://github.com/hetzneronline/community-content/blob/master/contributing.md#external-references). 

The guidelines state:
> If your tutorial includes products or software of a third party, make sure they are **free-of-charge and open-source**, so that everyone can follow your tutorial.

While aaPanel offers a [Basic Free plan](https://www.aapanel.com/new/pricing.html), it is not open-source.

image

Also you should include the following statement on your PR:
> I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: YOUR NAME <YOUR@EMAILPROVIDER.TLD>

**Comment by Akashs8383** [comment]
This tutorial is about aapanel free plan

**Comment by svenja11** [comment]
Hello @Akashs8383
I looked at your tutorial and @ShadowArcanist is right. Unfortunately, your tutorial does not qualify to be published at [community.hetzner.com](https://community.hetzner.com/tutorials). Mainly because we already have a tutorial that explains [how to install aaPanel](https://community.hetzner.com/tutorials/installing-aapanel) (will be updated soon). But it is also doesn't really provide additional information compared to the [official documentation](https://www.aapanel.com/new/download.html).

Hetzner Tutorial Guidelines:

> **Topic**
> * Before you start writing your tutorial, make sure there isn't already a tutorial at [community.hetzner.com](https://community.hetzner.com/tutorials) about the exact same topic.

**Comment by Akashs8383** [comment]
Thank you for the clarification.

**Pull request closed without merging**